docs/b2b-edition/specs/storefront/storefront/company.yaml

```diff
@@ -1,87 +1,77 @@
 openapi: 3.0.0
 info:
-  title: Company
-  version: '1.0'
-  description: Company refers to a collection of customers. BigCommerce B2B Edition special provided company member management and convenience of company purchasing.
+  title: Companies and Users
+  description: |-
+    ## Introduction
+
+    B2B Edition Companies are business accounts used by your clients to access their specific instance of the Buyer Portal. The queries and mutations in the Companies Storefront API allow you to:
```

Comment:
I'm not sure I'd specifically define companies in reference to the Buyer Portal.  It would be more appropriate to keep it more generic.